support/cas-server-support-oidc/src/test/java/org/apereo/cas/oidc/token/OidcIdTokenGeneratorServiceTests.java

```diff
@@ -76,9 +81,11 @@ public void verifyTokenGeneration() throws Exception {
 
         val claims = oidcTokenSigningAndEncryptionService.decode(idToken, Optional.ofNullable(registeredService));
         assertNotNull(claims);
-        assertTrue(claims.hasClaim("mail"));
-        assertEquals("casuser@example.org", claims.getStringClaimValue("mail"));
-        assertEquals(principal.getAttributes().get("cn"), claims.getStringListClaimValue("cn"));
+        assertTrue(claims.hasClaim(OIDC_CLAIM_EMAIL));
+        assertTrue(claims.hasClaim(OIDC_CLAIM_NAME));
+        assertFalse(claims.hasClaim(OIDC_CLAIM_PHONE_NUMBER));
+        assertEquals("casuser@example.org", claims.getStringClaimValue(OIDC_CLAIM_EMAIL));
+        assertEquals(principal.getAttributes().get(OIDC_CLAIM_NAME), claims.getStringListClaimValue(OIDC_CLAIM_NAME));
```

Comment:
Please also verify that claims returned are single-valued claims. For example, why is name a list here?